build.fsx

```diff
@@ -234,7 +234,7 @@ let buildLibraryRust() =
     copyFiles libraryDir "*.toml" buildDir
     copyFiles sourceDir "*.rs" outDir
 
-    runInDir buildDir ("cargo build")
+    // runInDir buildDir ("cargo build")
```

Comment:
@ncave here is an interesting question - Do we even need to do this here? Currently, you build anyway when you run the tests (everything builds from source in Rust), so this makes getting feedback harder when using the rust lang server to view tests when altering the AST.

Reply:
@alexswan10k Yes and no, technically no, we don't have to have it, but it is a useful early quick check if the `fable-library-rust` is broken by itself. If that's what's throwing the Rust lang server in the VS Code extension, then we can keep it commented. I tend to use the older Rust extension as it works better for me, not sure why, which one do you use?

Reply:
I use rust-analyzer, which I believe is the community favourite. I believe the official one has less features, and has some shortcomings. I have found rust-analyzer to work best for me.

I am more than happy to add this back in if you like, I was just trying to trim some fat :)